### PR TITLE
New pirate ship, now pirates will be skeletons

### PR DESCRIPTION
**Pull request by nicbn**
[Changelogs]: #
:cl:
add: Pirates got a new ship.
/:cl:

obiously-not-in-scale

### CODE REVIEW

**Comment by coiax** [comment]
...this looks like the old ship?

**Comment by nicbn** [comment]
Check #31947 for the old one.

File: _maps/templates/pirate_ship.dmm [hunk 1]
```diff
@@ -1016,125 +535,166 @@
...
-/obj/machinery/microwave{
-	pixel_y = 5
+/obj/item/melee/transforming/energy/sword/pirate,
+/obj/item/melee/transforming/energy/sword/pirate{
+	pixel_y = 12
+	},
+/obj/item/melee/transforming/energy/sword/pirate{
+	pixel_x = 12;
+	pixel_y = 7;
+	pixel_z = 0
```
Comment (ShizCalev) [inline _maps/templates/pirate_ship.dmm]: uh

File: _maps/templates/pirate_ship.dmm [hunk 2]
```diff
@@ -1016,125 +535,166 @@
...
-/obj/item/lighter,
-/obj/effect/decal/cleanable/dirt,
-/turf/open/floor/plasteel/bar,
+"bx" = (
+/obj/structure/sign/securearea{
+	desc = "A warning sign which reads 'EXTERNAL AIRLOCK'";
+	icon_state = "space";
+	layer = 4;
+	name = "EXTERNAL AIRLOCK";
+	pixel_x = 0;
```
Comment (ShizCalev) [inline _maps/templates/pirate_ship.dmm]: DV

File: _maps/templates/pirate_ship.dmm [hunk 3]
```diff
@@ -1016,125 +535,166 @@
...
-"cB" = (
-/obj/structure/chair/stool/bar,
-/obj/machinery/atmospherics/pipe/simple/supply/hidden,
-/turf/open/floor/plasteel/bar,
+"by" = (
+/obj/machinery/button/door{
+	id = "piratestarboardexternal";
+	name = "External Bolt Control";
+	normaldoorcontrol = 1;
+	pixel_x = 0;
```
Comment (ShizCalev) [inline _maps/templates/pirate_ship.dmm]: DV

File: _maps/templates/pirate_ship.dmm [hunk 4]
```diff
@@ -1149,129 +709,185 @@
...
+/obj/structure/table,
+/obj/item/stack/sheet/metal{
+	pixel_x = -1
 	},
-/obj/structure/cable/yellow{
-	d2 = 8;
-	icon_state = "0-8"
+/obj/item/stack/sheet/glass{
+	amount = 50;
+	pixel_x = 0;
```
Comment (ShizCalev) [inline _maps/templates/pirate_ship.dmm]: DV

File: _maps/templates/pirate_ship.dmm [hunk 5]
```diff
@@ -1149,129 +709,185 @@
...
-	dir = 8
+"bF" = (
+/obj/structure/table,
+/obj/item/stack/sheet/metal{
+	pixel_x = -1
 	},
-/obj/structure/cable/yellow{
-	d2 = 8;
-	icon_state = "0-8"
+/obj/item/stack/sheet/glass{
```
Comment (ShizCalev) [inline _maps/templates/pirate_ship.dmm]: glass/fifty

File: _maps/templates/pirate_ship.dmm [hunk 6]
```diff
@@ -1149,129 +709,185 @@
...
+	amount = 50;
```
Comment (ShizCalev) [inline _maps/templates/pirate_ship.dmm]: remove

File: _maps/templates/pirate_ship.dmm [hunk 7]
```diff
@@ -1149,129 +709,185 @@
...
-/turf/open/floor/plasteel,
+/obj/effect/decal/cleanable/dirt,
+/obj/effect/decal/cleanable/dirt,
+/turf/open/floor/wood,
+/area/shuttle/pirate)
+"bW" = (
+/obj/structure/table/reinforced,
+/obj/item/book/manual/barman_recipes{
+	pixel_x = -4;
+	pixel_y = 0
```
Comment (ShizCalev) [inline _maps/templates/pirate_ship.dmm]: DV

File: _maps/templates/pirate_ship.dmm [hunk 8]
```diff
@@ -421,246 +203,126 @@
...
-	pixel_y = 3
-	},
-/obj/item/reagent_containers/glass/beaker{
-	pixel_x = 3
+/obj/machinery/button/door{
+	id = "pirateportexternal";
+	name = "External Bolt Control";
+	normaldoorcontrol = 1;
+	pixel_x = -24;
+	pixel_y = 0;
```
Comment (ShizCalev) [inline _maps/templates/pirate_ship.dmm]: DV

File: _maps/templates/pirate_ship.dmm [hunk 9]
```diff
@@ -790,17 +393,32 @@
...
+/area/shuttle/pirate)
+"aW" = (
+/obj/effect/decal/cleanable/dirt,
+/obj/structure/sign/securearea{
+	desc = "A warning sign which reads 'EXTERNAL AIRLOCK'";
+	icon_state = "space";
+	layer = 4;
+	name = "EXTERNAL AIRLOCK";
+	pixel_x = -32;
+	pixel_y = 0
```
Comment (ShizCalev) [inline _maps/templates/pirate_ship.dmm]: DV

File: _maps/templates/pirate_ship.dmm [hunk 10]
```diff
@@ -1016,125 +535,166 @@
...
-"cx" = (
-/obj/machinery/airalarm{
-	pixel_y = 24;
+"bu" = (
+/obj/structure/chair/stool,
+/obj/machinery/power/apc{
+	dir = 4;
+	name = "Pirate Ship APC";
+	pixel_x = 26;
+	pixel_y = 0;
```
Comment (ShizCalev) [inline _maps/templates/pirate_ship.dmm]: DV

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Comment by praisenarsie** [comment]
looks more old

**Comment by nicbn** [comment]
It is smaller, has less room separation, uses a simpler engine and looks more like a pirate ship would.

**Comment by nicbn** [comment]
Oh it has no atmos too, because pirates don't need air

**Comment by XDTM** [comment]
Why skeletons though? Skeletons should kinda be limited to magic stuff, if you really want a nonbreathing race you might want androids instead

**Comment by nicbn** [comment]
https://www.youtube.com/watch?v=gZ_qJ7esLZo

**Comment by JJRcop** [comment]
I agree with @XDTM

**Comment by GunHog** [comment]
This one feels too simple and empty, I would rather we keep the current one.

**Comment by davethwave** [comment]
perhaps keep it but make it a Halloween event only?

**Comment by Xhuis** [comment]
I like this one a bit more. WJ has a thing for paramilitary supercruisers for any kind of spaceship, which is fine, but a state-of-the-art warship isn't generally something found in the hands of renegade space pirates. This one feels much more thematically fitting.

Skeletons are amusing but I'm kind of lost as to their point. I get the pun on "skeleton crew", but I dunno...

**Comment by nicbn** [comment]
The empty space is so that pirates can build their own things if they need to. Also skeletons are culturally related to pirates. Take, for example, the pirate flag, which is a skull.

**Comment by nicbn** [comment]
I gonna split the "pirates are now skeletons" to another PR

**Comment by Repukan** [comment]
Do you really need the hardsuits if skeles are EVA proof?

**Comment by nicbn** [comment]
Good point

**Comment by nicbn** [comment]
I'll rethink the design

**Pull request closed without merging**